src/shrutiDJun20/CompareIntArray.java

```diff
@@ -0,0 +1,32 @@
+/*
+ Find first differ index from given arrays. 
+arr1: {10,2,9,14,3}
+arr2: {10,2,18,14,3}
+o/p:- Values are not matching at index -> 2 
+ */
+package shrutiDJun20;
+
+public class CompareIntArray {
+
+	void compareArray(int[] input1, int[] input2){
```

Comment:
If you want to compare whole array then why you have a use a return statement. From a given method name It sounds like what I explained.

Reply:
changed method name and added else statement too